File: Problem 1/Solution.scala
// Define function with input Range and output Integer
def sumMultiplesOf3Or5(range: Range): Int = {
    // Filter all n in range such that only n mod 3 = 0 or n mod 5 = 0 are selected
    range.filter(n => n % 3 == 0 || n % 5 == 0).sum
}
// Define maximum value
val max = 1000;
// Print output 
println(s"The sum of all the multiples of 3 or 5 below ${max}: " + sumMultiplesOf3Or5(1 until max));

File: Problem 1/Solution.py
# Create 3 lists (multiples of 3, 5 and 15)
X = list(range(0,1000,3))
Y = list(range(0,1000,5))
Z = list(range(0,1000,15))

# Sum multiples of 3 and 5 while substracting multiples of 15
sum(X)+sum(Y)-sum(Z)